packages/validator/src/services/indices.ts

```diff
@@ -110,6 +116,49 @@ export class IndicesService {
         newIndices.push(validatorState.index);
       }
     }
+
+    logValidatorStatuses(allValidators, this.logger);
+
     return newIndices;
   }
 }
+
+function logValidatorStatuses(allValidators: Map<ValidatorStatus, ValidatorResponse[]>, logger: Logger): void {
+  const loggedValidatorStates: {[key: string]: ValidatorStatus[]} = {
+    "Validator activated": ["active"],
+    "Validator pending": ["pending_initialized", "pending_queued"],
+    "Validator withdrawn": ["withdrawal_done"],
+    "Validator exited": ["exited_slashed", "exited_unslashed"],
+  };
+
+  const logFormatter = (message: string, validatorState: ValidatorResponse): void => {
+    logger.info(message, {
+      pubkey: toHexString(validatorState.validator.pubkey),
+      index: validatorState.index,
+    });
+  };
+
+  for (const [message, statuses] of Object.entries(loggedValidatorStates)) {
+    getValidatorStatesByStatuses(statuses, allValidators).forEach((validatorState) => {
+      logFormatter(message, validatorState);
+    });
+  }
+
+  logger.info("Initialized validators", {
+    total: allValidators.size,
+    active: allValidators.get("active")?.length ?? 0,
+  });
+}
```

Comment:
We should avoid printing one line per validator here.
Can you refactor the one log line to be:
```ts
logger.info("Validator statuses", {
  active: 5,
  pending: 4,
  exited: 4,
  withdrawn: 3,
  total: 16,
});
```
Be sure to only print the each status total if the status is non-zero.

Reply:
shouldn't we show `total` first?